src/CMakeLists.txt

```diff
@@ -81,11 +80,12 @@ target_link_libraries(objc-metadata-generator
 
 set_target_properties(objc-metadata-generator PROPERTIES
     COMPILE_FLAGS "-Werror -Wall -Wextra -Wno-unused-parameter"
+    XCODE_ATTRIBUTE_GCC_SYMBOLS_PRIVATE_EXTERN YES
```

Comment:
Doesn't this work only for the Xcode generator? I believe we currently build the metadata generator using the Makefile generator.